.github/workflows/pages-deploy.yml.hook

```diff
@@ -54,17 +54,14 @@ jobs:
           bundle exec jekyll b
 
           if [[ -n $SPEC_TEST ]]; then
-            # Bypass the defects of htmlproofer
+            # build in a subdir (_site/baseurl) so it will pass the
+            # htmlproofer test
             bundle exec jekyll b -b "" -d "$SPEC_TEST"
           fi
```

Comment:
2. we can reduce one build:

```diff
-          bundle exec jekyll b
-
-          if [[ -n $SPEC_TEST ]]; then
-            # build in a subdir (_site/baseurl) so it will pass the
-            # htmlproofer test
-           bundle exec jekyll b -b "" -d "$SPEC_TEST"
-         fi
+        bundle exec jekyll b -d "_site$BASE_URL"
```